### DISCUSSION TITLE
Clipboard history won't work after using winutil.

### DISCUSSION THREAD

**Discussion by MeaniePoopoo** [ChrisTitusTech/winutil#1890] [Q&A] [answered]
![image](https://github.com/ChrisTitusTech/winutil/assets/155614848/8abcbeb4-8812-402a-99d4-c78945e888c7)
Idk why it's grayed out, it's still enabled but WIN + v doesn't work.

All I did in winutil was update some apps and ran the recommended laptop tweaks.
Attempting to fix this problem; I ran Undo Tweaks on Disable Telemetry but that didn't seem to fix the issue.

I also tried everything in this article:
https://www.elevenforum.com/t/enable-or-disable-clipboard-history-in-windows-11.973/
but WIN + v still shows nothing.

**Comment by 659dean** [comment, marked as answer]
I had the same issue - it's a default setting in OO ShutUp10

To fix, turn the two clipboard history off in OO by: Launch WinUtil > Tweaks > Customise OO Shutup Teaks > Disable "Disable storage of clipboard history" (under the 'Current User' Tab) > Select 'Local Machine' tab > disable "Disable storage of clipboard history for whole machine" > Exit & reset

**Reply by MeaniePoopoo** [reply]
It worked, thank you!

**Reply by xaiyaloun** [reply]
Thanks, I'm about to reinstall windows.
